src/index.js

```diff
@@ -4,7 +4,7 @@ import { BrowserRouter } from 'react-router-dom';
 import { Provider } from 'react-redux';
 import store from './redux/configureStore';
 import App from './App';
-import './index.scss';
+import './style/index.scss';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
```

Comment:
- [ ] Good job so far. However, kindly add all the placeholder from the Zeplin design because it's a requirement that your design match that of Zeplin, even if they are not functional. They should just be a placeholder but must be present in your app. Kindly match the Zeplin design.


Your Implementation | Zeplin Design
:---:|:---:
![](https://i.imgur.com/OizQdUJ.png) | ![](https://i.imgur.com/EzrdIjG.png)

Reply:
Ok! Thank you,